package.json

```diff
@@ -146,6 +146,7 @@
     "@types/iframe-resizer": "^3.5.9",
     "@types/inflection": "^1.13.0",
     "@types/isomorphic-fetch": "^0.0.35",
+    "@types/jest": "^28.1.1",
```

Comment:
`expect.closeTo` type is added in this version 🤦 . It wasn't in `"@types/jest": "^27.5.1"`